packages/block-library/src/navigation/style.scss

```diff
@@ -111,10 +111,16 @@
 	}
 
 	@include break-small() {
-		display: flex;
-		flex-direction: row;
-		position: relative;
-		background-color: inherit;
+		&:not(.is-menu-open) {
```

Comment:
Oh nice, if this does what I think it does, clever!

Reply:
It applies the desktop style for the menu only if the modal has been explicitly closed by the user!

Reply:
So we can keep the has-modal-open? 

👌